clients/cloud/rust/README.md

```diff
@@ -0,0 +1,61 @@
+# Overview
+
+Produce messages to produce and consume messages from [Confluent Cloud](https://www.confluent.io/confluent-cloud/) using the [rust-rdkafka client for Apache Kafka](https://github.com/fede1024/rust-rdkafka).
+
+# Prerequisites
+* [Rust client for Apache Kafka](https://github.com/fede1024/rust-rdkafka#installation) installed on your machine
+* Access to a [Confluent Cloud](https://www.confluent.io/confluent-cloud/) cluster
+* Local file with configuration parameters to connect to your Confluent Cloud instance ([how do I find those?](https://docs.confluent.io/current/cloud/using/config-client.html#librdkafka-based-c-clients)). Format the file as follows:
+
+```bash
+$ cat ~/.ccloud/example.config
+bootstrap.servers=<broker-1,broker-2,broker-3>
+sasl.username=<api-key-id>
+sasl.password=<secret-access-key>
+```
+
+# Example 1: Hello World!
+In this example, the producer writes Kafka data to a topic in Confluent Cloud. 
+Each record has a key representing a username (e.g. `alice`) and a value.
+The consumer reads the same topic from Confluent Cloud.
+
+1. Run the producer, passing in arguments for (a) the local file with configuration parameters to connect to your Confluent Cloud instance and (b) the topic name:
+
+```bash
+$ RUSTFLAGS='-C link-args=-lzstd' cargo build
+$ ./target/debug/producer --config ~/.ccloud/example.conf --topic test1
+Preparing to produce record: alice 0
+Preparing to produce record: alice 1
+Preparing to produce record: alice 2
+Preparing to produce record: alice 3
+Preparing to produce record: alice 4
+Preparing to produce record: alice 5
+Preparing to produce record: alice 6
+Preparing to produce record: alice 7
+Preparing to produce record: alice 8
+Successfully produced record to topic test1 partition [5] @ offset 18
+Successfully produced record to topic test1 partition [5] @ offset 19
+Successfully produced record to topic test1 partition [5] @ offset 20
+Successfully produced record to topic test1 partition [5] @ offset 21
+Successfully produced record to topic test1 partition [5] @ offset 22
+Successfully produced record to topic test1 partition [5] @ offset 23
+Successfully produced record to topic test1 partition [5] @ offset 24
+Successfully produced record to topic test1 partition [5] @ offset 25
+Successfully produced record to topic test1 partition [5] @ offset 26
+```
+
+2. Run the consumer, passing in arguments for (a) the local file with configuration parameters to connect to your Confluent Cloud instance and (b) the same topic name as used above. Verify that the consumer received all the messages:
+
+```bash
+$ RUSTFLAGS='-C link-args=-lzstd' cargo build
```

Comment:
This repeats the build step. Maybe put the build as a separate step before running the producer and cosnumer?

Reply:
see above comment

Reply:
oh I see what you mean - yea you're right this step is redundant